Backpack/Theme/Classes/BPKDohaTheme.m

```diff
@@ -52,27 +53,31 @@ - (UIColor *)buttonLinkContentColor {
 }
 
 - (UIColor *)buttonPrimaryContentColor {
-    return [UIColor colorWithRed:6.0f / 255.0f green:96.0f / 255.0f blue:96.0f / 255.0f alpha:1.0f];
+    return BPKColor.white;
```

Comment:
The reason I made these different to the default is that we want to be able to see it working!

Reply:
We don't do that on Web, we should consider an alternative method I think.